be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -304,7 +307,10 @@ Status KafkaDataConsumer::get_partition_offset(std::vector<int32_t>* partition_i
         if (err != RdKafka::ERR_NO_ERROR) {
             LOG(WARNING) << "failed to query watermark offset of topic: " << _topic << " partition: " << p_id
                          << ", err: " << RdKafka::err2str(err);
-            return Status::InternalError("failed to query watermark offset, err: " + RdKafka::err2str(err));
+            std::stringstream ss;
+            ss << "failed to query watermark offset, err: " << RdKafka::err2str(err);
```

Comment:
Failed to query Kafka watermark offset? (and put it into error_code.h/cpp)
Is the watermark a common word for Kafka? can we just use "Failed to query Kafka offsets"?